HornsAndHooves-flat_map.gemspec

```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # specify any dependencies here; for example:
-  s.add_dependency(%q<activesupport>, ["~> 4.2"])
-  s.add_dependency(%q<activerecord>, ["~> 4.2"])
-  s.add_dependency(%q<yard>, [">= 0"])
+  s.add_dependency "activesupport", "> 4.2", "< 6.0.0"
```

Comment:
Why can't this be `>= 4.2`?

Reply:
That's what I started with and I chickened out with the rationale that I didn't test it with 6.

Reply:
No, I mean `">= 4.2", "< 6.0.0"`